sdk/keyvault/azure_security_keyvault_keys/src/generated/clients/key_vault_client.rs

```diff
@@ -0,0 +1,827 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) Rust Code Generator. DO NOT EDIT.
+
+use azure_core::{
+    BearerTokenCredentialPolicy, ClientMethodOptions, ClientOptions, Context, Method, Pager, Pipeline, Policy, Request, RequestContent, Response, Result, Url,
+};
+use azure_core::credentials::TokenCredential;
+use crate::models::{
+    BackupKeyResult, DeletedKeyBundle, DeletedKeyListResult, GetRandomBytesRequest, KeyBundle, KeyCreateParameters, KeyImportParameters, KeyListResult, KeyOperationResult, KeyOperationsParameters, KeyReleaseParameters, KeyReleaseResult, KeyRestoreParameters, KeyRotationPolicy, KeySignParameters, KeyUpdateParameters, KeyVerifyParameters, KeyVerifyResult, RandomBytes,
+};
+use std::sync::Arc;
+use typespec_client_core::json;
+use typespec_client_core::fmt::SafeDebug;
+use typespec_client_core::http::PagerResult;
+
+pub struct KeyVaultClient {
```

Comment:
@RickWinter this has to be `KeyClient`, along with the options. You'll need a `client.tsp` update like you did with Secrets.

Reply:
Updated in the TypeSpec client.tsp